specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -23,4 +37,91 @@
 
 ## Proving Fault using ZKVM
 
+### Prover
+
+A prover plays a vital role in the challenge process, acting as an integral part of the challenger responsible
+for generating proof for the target block. This proof substantiates claims regarding the legitimacy of block data
+and state transitions.
```

Comment:
I'm not sure if Prover part is necessary for this section, Proving Fault using ZKVM. How about changing like below?
```suggestion
### ZK Fault Proof
```